enable_image_fetching default value change

barrust opened the pull request:
* Change default value for `enable_image_fetching` to be `False` #31 
* Update Documentation 
    * Add Video and Image classes
* Re-enable missing image tests

Enkerli commented:
Might also make sense to make things clear in terms of this option (as well as `strict`). Maybe mention them in the [Configuration section of the Readme](https://github.com/goose3/goose3/blob/master/README.rst#configuration)?

barrust commented:
I agree; thank you for pointing that out. I updated the readme to show more information about using those specific configuration options. 

Also, I believe that the `known_context_patterns` is order sensitive so instead of appending new ones to the end, I changed it to add them to the beginning of the list. We may want a method to append to the beginning or the end of the list. wouldn't be difficult to set up.
